src/EditorFeatures/Core.Wpf/InlineRename/AbstractEditorInlineRenameService.cs

```diff
@@ -217,8 +217,8 @@ private IInlineRenameInfo GetRenameInfo(Document document, int position, Cancell
             }
 
             return new SymbolInlineRenameInfo(
-                refactorNotifyServices, document, triggerToken.Span, 
-                symbolAndProjectId, forceRenameOverloads, cancellationToken);
+                refactorNotifyServices, document, triggerToken.Span,
```

Comment:
since you give in document. you can always get ISyntaxFactsService from document.GetLanguageService<ISyntaxFactsService>() and call .IsCaseSensitive rather than pass that info in.